### PR TITLE
(DOCSP-16646) Backfills docs content for atlas dataLakes commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for dataLakes commands and updates descriptions and flags to reference Atlas Data Federation

_Jira ticket:_
https://jira.mongodb.org/browse/DOCSP-16646
https://jira.mongodb.org/browse/DOCSP-16647
https://jira.mongodb.org/browse/DOCSP-16648
https://jira.mongodb.org/browse/DOCSP-16649
https://jira.mongodb.org/browse/DOCSP-16650

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works **N/A**
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added) **N/A**
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/atlascli/command/atlas-dataLakes-create.txt [hunk 1]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to create.
+     - Name of the Atlas federated database to create.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-dataLakes-create.txt:42]: We should refer to the product as Atlas Data Federation and the virtual databases as federated database instances. So, suggest:
```suggestion
     - Name of the federated database instance to create.
```

File: docs/atlascli/command/atlas-dataLakes-delete.txt [hunk 2]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to delete.
+     - Name of the Atlas federated database instance to delete.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-dataLakes-delete.txt:42]:
```suggestion
     - Name of the federated database instance to delete.
```

File: docs/atlascli/command/atlas-dataLakes-describe.txt [hunk 3]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to retrieve.
+     - Name of the Atlas federated database instance to retrieve.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-dataLakes-describe.txt:42]:
```suggestion
     - Name of the federated database instance to retrieve.
```

File: docs/atlascli/command/atlas-dataLakes-list.txt [hunk 4]
```diff
@@ -12,7 +12,9 @@ atlas dataLakes list
    :depth: 1
    :class: singlecol
 
-List Atlas Data Lakes for your project.
+List all Atlas federated database instances for your project.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-dataLakes-list.txt:15]:
```suggestion
List all federated database instances for your project.
```

File: docs/atlascli/command/atlas-dataLakes-update.txt [hunk 5]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to update.
+     - Name of the Atlas federated database instance to update.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-dataLakes-update.txt:42]:
```suggestion
     - Name of the federated database instance to update.
```

File: docs/atlascli/command/atlas-dataLakes.txt [hunk 6]
```diff
@@ -51,11 +51,11 @@ Inherited Options
...
 
-* :ref:`atlas-dataLakes-create` - Create a new data lake for your project.
-* :ref:`atlas-dataLakes-delete` - Delete a data lake from your project.
-* :ref:`atlas-dataLakes-describe` - Return a specific data lake.
-* :ref:`atlas-dataLakes-list` - List Atlas Data Lakes for your project.
-* :ref:`atlas-dataLakes-update` - Update a data lake for your project.
+* :ref:`atlas-dataLakes-create` - Create a new federated database instance for your project.
+* :ref:`atlas-dataLakes-delete` - Delete a federated database instance from your project.
+* :ref:`atlas-dataLakes-describe` - Return the details for the specified federated database instance.
+* :ref:`atlas-dataLakes-list` - List all Atlas federated database instances for your project.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-dataLakes.txt:57]:
```suggestion
* :ref:`atlas-dataLakes-list` - List all federated database instances for your project.
```

File: docs/mongocli/command/mongocli-atlas-dataLakes-create.txt [hunk 7]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to create.
+     - Name of the Atlas federated database to create.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-dataLakes-create.txt:42]:
```suggestion
     - Name of the federated database to create.
```

File: docs/mongocli/command/mongocli-atlas-dataLakes-delete.txt [hunk 8]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to delete.
+     - Name of the Atlas federated database instance to delete.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-dataLakes-delete.txt:42]:
```suggestion
     - Name of the federated database instance to delete.
```

File: docs/mongocli/command/mongocli-atlas-dataLakes-describe.txt [hunk 9]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to retrieve.
+     - Name of the Atlas federated database instance to retrieve.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-dataLakes-describe.txt:42]:
```suggestion
     - Name of the federated database instance to retrieve.
```

File: docs/mongocli/command/mongocli-atlas-dataLakes-list.txt [hunk 10]
```diff
@@ -12,7 +12,9 @@ mongocli atlas dataLakes list
    :depth: 1
    :class: singlecol
 
-List Atlas Data Lakes for your project.
+List all Atlas federated database instances for your project.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-dataLakes-list.txt:15]:
```suggestion
List all federated database instances for your project.
```

File: docs/mongocli/command/mongocli-atlas-dataLakes-update.txt [hunk 11]
```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to update.
+     - Name of the Atlas federated database instance to update.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-dataLakes-update.txt:42]:
```suggestion
     - Name of the federated database instance to update.
```

File: docs/mongocli/command/mongocli-atlas-dataLakes.txt [hunk 12]
```diff
@@ -51,11 +51,11 @@ Inherited Options
...
 
-* :ref:`mongocli-atlas-dataLakes-create` - Create a new data lake for your project.
-* :ref:`mongocli-atlas-dataLakes-delete` - Delete a data lake from your project.
-* :ref:`mongocli-atlas-dataLakes-describe` - Return a specific data lake.
-* :ref:`mongocli-atlas-dataLakes-list` - List Atlas Data Lakes for your project.
-* :ref:`mongocli-atlas-dataLakes-update` - Update a data lake for your project.
+* :ref:`mongocli-atlas-dataLakes-create` - Create a new federated database instance for your project.
+* :ref:`mongocli-atlas-dataLakes-delete` - Delete a federated database instance from your project.
+* :ref:`mongocli-atlas-dataLakes-describe` - Return the details for the specified federated database instance.
+* :ref:`mongocli-atlas-dataLakes-list` - List all Atlas federated database instances for your project.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-dataLakes.txt:57]:
```suggestion
* :ref:`mongocli-atlas-dataLakes-list` - List all federated database instances for your project.
```

File: internal/cli/atlas/datalake/create.go [hunk 13]
```diff
@@ -74,11 +75,14 @@ func CreateBuilder() *cobra.Command {
 	opts := &CreateOpts{}
 	cmd := &cobra.Command{
 		Use:   "create <name>",
-		Short: "Create a new data lake for your project.",
+		Short: "Create a new federated database instance for your project.",
+		Long:  "To learn more about Atlas Data Federation (previously named Atlas Data Lake), see https://www.mongodb.com/docs/atlas/data-federation/overview/.",
 		Args:  require.ExactArgs(1),
 		Annotations: map[string]string{
-			"nameDesc": "Name of the Atlas Data Lake to create.",
+			"nameDesc": "Name of the Atlas federated database to create.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/datalake/create.go:82]:
```suggestion
			"nameDesc": "Name of the federated database instance to create.",
```

File: internal/cli/atlas/datalake/delete.go [hunk 14]
```diff
@@ -52,11 +53,14 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <name>",
 		Aliases: []string{"rm"},
-		Short:   "Delete a data lake from your project.",
+		Short:   "Delete a federated database instance from your project.",
+		Long:    "To learn more about Atlas Data Federation (previously named Atlas Data Lake), see https://www.mongodb.com/docs/atlas/data-federation/overview/.",
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"nameDesc": "Name of the Atlas Data Lake to delete.",
+			"nameDesc": "Name of the Atlas federated database instance to delete.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/datalake/delete.go:60]:
```suggestion
			"nameDesc": "Name of the federated database instance to delete.",
```

File: internal/cli/atlas/datalake/describe.go [hunk 15]
```diff
@@ -59,11 +60,14 @@ func DescribeBuilder() *cobra.Command {
 	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
 		Use:   "describe <name>",
-		Short: "Return a specific data lake.",
+		Short: "Return the details for the specified federated database instance.",
+		Long:  "To learn more about Atlas Data Federation (previously named Atlas Data Lake), see https://www.mongodb.com/docs/atlas/data-federation/overview/.",
 		Args:  require.ExactArgs(1),
 		Annotations: map[string]string{
-			"nameDesc": "Name of the Atlas Data Lake to retrieve.",
+			"nameDesc": "Name of the Atlas federated database instance to retrieve.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/datalake/describe.go:67]:
```suggestion
			"nameDesc": "Name of the federated database instance to retrieve.",
```

File: internal/cli/atlas/datalake/list.go [hunk 16]
```diff
@@ -59,9 +60,12 @@ func ListBuilder() *cobra.Command {
 	opts := &ListOpts{}
 	cmd := &cobra.Command{
 		Use:     "list",
-		Short:   "List Atlas Data Lakes for your project.",
+		Short:   "List all Atlas federated database instances for your project.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/datalake/list.go:63]:
```suggestion
		Short:   "List all federated database instances for your project.",
```

File: internal/cli/atlas/datalake/update.go [hunk 17]
```diff
@@ -88,11 +89,14 @@ func UpdateBuilder() *cobra.Command {
 	opts := &UpdateOpts{}
 	cmd := &cobra.Command{
 		Use:   "update <name>",
-		Short: "Update a data lake for your project.",
+		Short: "Modify a federated database instance for your project.",
+		Long:  "To learn more about Atlas Data Federation (previously named Atlas Data Lake), see https://www.mongodb.com/docs/atlas/data-federation/overview/.",
 		Args:  require.ExactArgs(1),
 		Annotations: map[string]string{
-			"nameDesc": "Name of the Atlas Data Lake to update.",
+			"nameDesc": "Name of the Atlas federated database instance to update.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/datalake/update.go:96]:
```suggestion
			"nameDesc": "Name of the federated database instance to update.",
```

**Review by kanchana-mongodb — APPROVED** [review]
LGTM

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**